packages/playground/website/public/plugin-proxy.php

```diff
@@ -78,6 +78,12 @@ public function streamFromGithubPR($organization, $repo, $pr, $workflow_name, $a
                 continue;
             }
 
+            // If HTTP method is HEAD or OPTIONS, just say it's 200 OK
+            if ($_SERVER['REQUEST_METHOD'] === 'HEAD' || $_SERVER['REQUEST_METHOD'] === 'OPTIONS') {
+                header('HTTP/1.1 200 OK');
+                return;
+            }
```

Comment:
these requests can still 404 or 500 or 301 or 302…

this might have unintended consequences here. what's the goal for wiping out `HEAD` and `OPTIONS` requests?

Reply:
also some of these are important for CORS pre-flight requests showing what's allowable. will this break those?

Reply:
honestly I have no idea what's going on here or why we have these exclusions, so I don't understand the `&mode=verfiy` suggestion either

Reply:
Ah, I just need to a way to ask a question "can this Gutenberg PR be previewed in Playground?" The answer is typically yes if the PR exists and the GitHub CI provides a zipped plugin bundle. If the answer is no, however, I want to show an informative error message before redirecting the user to Playground.

Reply:
so the `HEAD` request was _from_ the Playground to see if the resource exists?
why did that not work? why were we forcing it as a 200?

or are you trying to perform an application level transform in the proxy layer? would it be easier if the app JS made the two requests? first a `HEAD` to see if the resource exists, and then a `GET` if it does?